aiohttp/web_response.py

```diff
@@ -362,7 +362,8 @@ def _start(self, request,
                 del headers[CONTENT_LENGTH]
         elif self._length_check:
             writer.length = self.content_length
-            if writer.length is None and version >= HttpVersion11:
+            if (writer.length is None and version >= HttpVersion11 and
```

Comment:
I think fix should be applied to `web_fileresponse.py` instead of base `StreamResponse` class.